### ISSUE TITLE
improve memory performance of transitfeed.py

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#30]
```
processing feeds takes tons of memory and it seems to get worse with each
release. Internally we have some large GTFS files that won't load on a 2GB
machine.

We need to do some profiling to work out if we are doing something that
prevents python doing the right thing.

It might be worth trying to reload a medium sized feed several times. If we
delete the Schedule object each time then the python process should stop
growing.

Here's someone with debugging a possibly similar problem
http://mail.python.org/pipermail/python-list/2007-June/445838.html
but he didn't get a reply.
```

Original issue reported on code.google.com by `tom.brow...@gmail.com` on 28 Jun 2007 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
I wasn't able to process my entire feed (52000 trips, 810000 stop times) in 
360MB of
memory. So I had to split it into 4 roughly even sized feeds, based on route. 
Then I
could load these into the validator and schedule viewer. Inconvenient, but I 
think
it's OK for the purposes of testing. Good luck in fixing this.
```

Original comment by `rodney.l...@gmail.com` on 29 May 2008 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
By the way, I have a fix under review (http://codereview.appspot.com/2644) in 
the
branch
http://googletransitdatafeed.googlecode.com/svn/branches/tom.brown.code-sqlite/p
ython
and put Windows binaries at
http://googletransitdatafeed.googlecode.com/files/transitfeed-sqlite.zip

This has been working for me.


```

Original comment by `tom.brow...@gmail.com` on 29 Jul 2008 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
I have released 1.1.6 which uses sqlite for the stop_times table. Slow, but 
works.
```

Original comment by `tom.brow...@gmail.com` on 7 Oct 2008 at 11:06
- Changed state: **Fixed**
